ccd-definition/CaseTypeTab/DefaultJudgmentDetailsDJSpec.json

```diff
@@ -1,24 +1,23 @@
 [
   {
     "CaseTypeID": "CIVIL",
```

Comment:
You cant do changes directly on production file. There should be prod versin for it. Could we keep same TabId?

Reply:
Done